_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Sharing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Sharing custom workloads
+
+You can share a custom workload with other OpenSearch users by uploading it to the [workloads repository](https://github.com/opensearch-project/opensearch-benchmark-workloads/) in GitHub. 
+
+Make sure that any data included in the workload's dataset does not contain any proprietary data or Personal Identifiable Information (PII). 
+
+To share a custom workload, follow these steps.
+
+## Create a README.md
+
+Provide a detailed `README.MD` file that includes the following:  
+
+- The purpose of the workload. When creating a description for the workload, consider the specific use case of the workload and how the workload's use case is different from other workloads included in the [workloads repository](https://github.com/opensearch-project/opensearch-benchmark-workloads/).
+- An example document from the dataset that helps users understand the data's structure.
+- The workload parameters that can be used to customize the workloads.
+- A list of default test procedures included in the workload and other test procedures that the workload can run.
+- An output sample that the workload produces after a test is run.
+- A copy of the open source license that gives the user and OpenSearch Benchmark permission to use the dataset.
+
+For an example workload README, go to the `http_logs` [README](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/http_logs/README.md).
+
+## Verify the workload's structure
+
+The workload must include the following files: 
+
+- `workload.json`
+- `index.json`
+- `files.txt`
+- `test_procedures/default.json`
+- `operations/default.json` 
+
+Both `default.json` file names can be customized to have a descriptive name. The workload can include an optional `workload.py` file to add more dynamic functionality. For more information about a file's contents, go to [Anatomy of a workload]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload/).
+
+## Testing the workload
+
+All workloads contributed to OpenSearch Benchmark must fulfil the following testing requirements: 
+
+- All tests run to explore the workload and produce example from the work must target an OpenSearch cluster.
+- The workload passes all integration tests. Follow these steps to ensure the workload passes the integration tests:
+   1. Add the workload to your forked copy of the [workloads repository](https://github.com/opensearch-project/opensearch-benchmark-workloads/).
+   2. In the forked repository, update the `benchmark-os-it.ini` and `benchmark-in-memory.ini` files in the `/osbenchmark/it/resources` directory to point to the forked workloads repository containing your workload.
+   3. After you've modified the `.ini` files, commit your changes to a branch for testing.
```

Comment:
3. After you've modified the `.ini` files, commit your changes to a branch for testing.  In addition to these two configuration files, you should include your new workload as well.  Merge your changes to the `1` and `2` branches, since OSB checks these branches out when it runs tests against OpenSearch v1 and v2 clusters as part of the integration tests.